### ISSUE TITLE
Add half-elliptical AoIs

### ISSUE THREAD

**Issue by angelaradulescu** [nivlab/NivLink#12]

**Comment by szorowi1** [comment]
Yes, you're absolutely right. Good catch and sorry for the slip-up!

Recreated the error with the following code:
```python 
from nivlink import ScreenInfo
%matplotlib inline
info = ScreenInfo(100,100,500)
info.add_rectangle_aoi(20,40,10,40)
info.add_rectangle_aoi(55,95,15,35)
info.add_ellipsoid_aoi(60,25,20,10)
info.add_ellipsoid_aoi(60,75,10,20)
info.plot_aoi(ticks=True)
```

The rectangles act as they should, but the ellipsoids are inverted (*x* position dictates *y* and vice-versa).

**Comment by angelaradulescu** [comment]
Sweet, thanks!!

**Comment by szorowi1** [comment]
Addressed under #14. Thanks, @angelaradulescu!
